src/View/Components/_footer.html.twig

```diff
@@ -0,0 +1,13 @@
+{% block stylesheet %}
```

Comment:
you don't need any stylesheet block inside footer view since you're importing it inside the layout